# Type: refactor
# Subtype: R3
# English Summary: Change variable access modifier to private
# Chinese Summary: 更改变量访问修饰符为private
# Entry Number: 11

# === Git Diff Content ===

diff --git a/java/org/apache/coyote/AbstractProtocol.java b/java/org/apache/coyote/AbstractProtocol.java
index 9bb52072c..455ba78b4 100644
--- a/java/org/apache/coyote/AbstractProtocol.java
+++ b/java/org/apache/coyote/AbstractProtocol.java
@@ -660,11 +660,10 @@ public abstract class AbstractProtocol<S> implements ProtocolHandler,
     protected static class ConnectionHandler<S> implements AbstractEndpoint.Handler<S> {
 
         private final AbstractProtocol<S> proto;
-
-        protected final RequestGroupInfo global = new RequestGroupInfo();
-        protected final AtomicLong registerCount = new AtomicLong(0);
-        protected final ConcurrentHashMap<S,Processor> connections = new ConcurrentHashMap<>();
-        protected final RecycledProcessors recycledProcessors = new RecycledProcessors(this);
+        private final RequestGroupInfo global = new RequestGroupInfo();
+        private final AtomicLong registerCount = new AtomicLong(0);
+        private final ConcurrentHashMap<S,Processor> connections = new ConcurrentHashMap<>();
+        private final RecycledProcessors recycledProcessors = new RecycledProcessors(this);
 
         public ConnectionHandler(AbstractProtocol<S> proto) {
             this.proto = proto;
